1228-public_page/user_navigates_to_landing_page_for_registration_and_can_see_files.feature

```diff
@@ -0,0 +1,51 @@
+Feature: User navigates to Landing Page for Registration and can see files
+
+    @1530
+    Scenario: Files that are Administrative Agreements are hidden
+        Given Anonymous User views Landing Page for Registration
+        And the Registration contains a File, which is an Administrative Agreement
+        When they view the Files section
+        Then they do not see the File that is an Administrative Agreement
+
+    Scenario: Files that are part of Registration are listed
+        Given Anonymous User views Landing Page for Registration
+        And the Registration contains Files
+        When they view the Files section
+        Then they can see files that are not Administrative Agreements are listed
+        And for each file they can see:
+            | Name    |
+            | Size    |
+            | Version |
+            | License |
+        And they can download Files that are not Embargoed
```

Comment:
Hva betyr dette? At det er en knapp for å laste ned filene?

Reply:
Ja. Er det bedre med `And they can see download button for Files that are not Embargoed`?

Reply:
Enig i at det er bedre å spesifisere at dette er en knapp